web/packages/teleport/src/services/audit/gen-event-reference/gen-event-reference.ts

```diff
@@ -5,30 +5,26 @@ import { Event, Formatters } from '../types';
 export function eventsWithoutExamples(
   fixtures: Event[],
   formatters: Formatters
-) {
-  const fixtureMap = new Map();
-  let result = [];
-  fixtures.forEach(f => {
-    fixtureMap.set(f.code, true);
-  });
-  Object.keys(formatters).forEach(k => {
-    if (fixtureMap.has(k)) {
-      return;
+): Event[] {
+  const fixtureCodes = new Set(fixtures.map(fixture => fixture.code));
+  return Object.keys(formatters).reduce((accum, current) => {
+    if (fixtureCodes.has(current)) {
+      return accum;
     }
-    result.push({
-      codeDesc: formatters[k].desc,
-      code: k,
+    accum.push({
+      codeDesc: formatters[current].desc,
+      code: current,
       raw: {
-        event: formatters[k].type,
+        event: formatters[current].type,
       },
     });
-  });
-  return result;
+    return accum;
+  }, []);
```

Comment:
```suggestion
  }, [] as Event[]);
```

You need to cast the initial value of the accumulator as TypeScript doesn't seem to be able to correctly infer it.